impl/src/planning.rs

```diff
@@ -63,6 +63,8 @@ struct DependencyNames {
   build_dep_names: Vec<String>,
   // Dependencies that are required for tests
   dev_dep_names: Vec<String>,
+  // Dependencies that have been renamed
+  renamed_dep_names: HashMap<String, String>,
```

Comment:
could this be called `aliased_dep_names`, and then `aliased_deps` line 80?

I think that will make it a little more clear that aliases are being used the entire time.

Reply:
Changed the names.